File: api/Bookshop.Application/Features/Books/Commands/Comments/CommentCommandResponse.cs
using Bookshop.Application.Features.Response;

namespace Bookshop.Application.Features.Books.Commands.Comments
{
    public class CommentCommandResponse : CommandResponse
    {
        public CommentCommandResponse() : base()
        {

        }
        public CommentResponseDto? Comment { get; set; }
    }
}

File: ui/src/app/dto/handler-response/book/comment/comment-command.response.ts
import { CommentResponseDto } from "../../../book/comment/comment-response-dto";
import { CommandResponse } from "../../../response/command-response";

export class CommentCommandResponse extends CommandResponse {
    comment: CommentResponseDto;

    constructor(message?: string, error?: boolean, validationErrors?: string[]) {
        super(message, error, validationErrors);
        this.comment = new CommentResponseDto();
    }
}